### PR TITLE
622 Add supported format types for standardization

### PR DESCRIPTION
**Pull request by Zejnilovic**

### CODE REVIEW

File: _docs/2.0.0/usage/standardization-formats.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    - usage
+---
+
+Currently Standardization supports these formats
+
+- [Cobol](#cobol) (see [Cobrix GitHub](https://github.com/AbsaOSS/cobrix/))
+- [CSV](#csv) (see [rfc4180](https://tools.ietf.org/html/rfc4180))
+- [FixedWidth](#fixed-width) (see `Link to be added repo does not exist yet`)
+- [JSON](json) (see [json.org](https://www.json.org/json-en.html))
+- [Parquet](parquet) (see [Apache Parquet](https://parquet.apache.org/documentation/latest/))
```
Comment (dk1844) [inline _docs/2.0.0/usage/standardization-formats.md:16]: Dont you mean `#json` & `#parquet` (in-file links?)

Comment (Zejnilovic) [reply _docs/2.0.0/usage/standardization-formats.md:16]: Good catch

File: _docs/2.0.0/usage/standardization-formats.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- [XML](#xml) (see [xml.com](https://www.xml.com/))
+
+To use any of the format you need to add to you standardization
```
Comment (benedeki) [inline _docs/2.0.0/usage/standardization-formats.md:19]:
```suggestion
When running standardization one of the formats of the list has to be specified.
```

File: _docs/2.0.0/usage/standardization-formats.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+| charset | Any valid charset names | The character set. By default `UTF-8`. |
+
+## CSV
+
+CSV `format` option is `csv`. Format options are
+
+| Option | Values | Description |
+|---|---|---|
+| header | Boolean | By default `false`. Specifies if the input data have a CSV style header |
+| delimiter | Any char or unicode such as `U+00A1` | By default `,`. Delimiter of the data on single row |
```
Comment (benedeki) [inline _docs/2.0.0/usage/standardization-formats.md:47]:
```suggestion
| delimiter | Any char or unicode such as `U+00A1` | By default `,`. Delimiter the column values on a row |
```

File: _docs/2.0.0/usage/standardization-formats.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+Has no extra options. Only `--format parquet`.
+
+## XML
+
+XML `format` option is `xml`. Format options are
+
+| Option | Values | Description |
+|---|---|---|
+| rowTag | String | The row tag of your xml files to treat as a row. For example, in this xml `<books> <book><book> ...</books>`, the appropriate value would be `book`. |
```
Comment (benedeki) [inline _docs/2.0.0/usage/standardization-formats.md:80]:
```suggestion
| rowTag | String | The tag of the xml file to treat as a row. For example, in the following xml `<books> <book><book> ...</books>`, the appropriate value would be `book`. |
```

File: _docs/2.0.0/usage/standardization-formats.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+Cobol `format` option is `cobol`. Format options are
+
+| Option | Values | Description |
+|---|---|---|
+| copybook | String | Path to a copybook for COBOL data format |
+| is-xcom | Boolean | Does a mainframe file in COBOL format contain XCOM record headers |
+| cobol-is-text | Boolean | Specifies if the mainframe file is ASCII text file |
+| cobol-encoding | `ascii` or `ebcdic` | Specify encoding of mainframe files |
+| cobol-trimming-policy | `none`, `left`, `right`, `both` | Specify string trimming policy for mainframe files |
+| charset | Any valid charset names | The character set. By default `UTF-8`. |
```
Comment (benedeki) [inline _docs/2.0.0/usage/standardization-formats.md:38]:
```suggestion
| charset | Any valid charset name | The character set of the input. By default `UTF-8`. |
```
Multiple times

File: _docs/2.0.0/usage/standardization-formats.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
+---
+layout: docs
+title: Usage - Standardization Formats
```
[the remark this answers is not in the document]
Comment (Zejnilovic) [reply _docs/2.0.0/usage/standardization-formats.md]: Used `Standardization Input Formats`

File: _docs/2.0.0/usage/standardization-formats.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+...standardization options...
+--format <format>
+--optionX valueY
+```
+
+## Cobol
+
+Cobol `format` option is `cobol`. Format options are
+
+| Option | Values | Description |
```
[the remark this answers is not in the document]
Comment (Zejnilovic) [reply _docs/2.0.0/usage/standardization-formats.md]: done

**Pull request merged**